src/main.rs

```diff
@@ -166,6 +164,7 @@ async fn execute_swap(sender: Sender<FinishedSwap>) -> anyhow::Result<()> {
 
             if let Err(e) = sender
                 .send(FinishedSwap::new(
+                    swap_id,
                     Free::Dai(beta_params.asset.into()),
```

Comment:
Hum, what is that? 🤔

Reply:
I did find the name a bit confusing if that's what you mean.

Reply:
I think for the purposes of writing it to the history file you will need to modify this.